src/Microsoft.Health.Fhir.Api/Modules/FhirModule.cs

```diff
@@ -144,6 +162,10 @@ public void Load(IServiceCollection services)
 
             services.AddSingleton<INarrativeHtmlSanitizer, NarrativeHtmlSanitizer>();
 
+            var stu3ModelFactory = new Stu3ModelFactory();
+            services.Add(c => stu3ModelFactory).Singleton().AsSelf().AsImplementedInterfaces();
```

Comment:
nit: can we keep the parameter name consistent across (it's showing up as `c`, `x`, and `provider` in the same file).